qiita_db/util.py

```diff
@@ -941,20 +1002,21 @@ def get_lat_longs():
     list of [float, float]
         The latitude and longitude for each sample in the database
     """
-    conn = SQLConnectionHandler()
-    sql = """SELECT DISTINCT table_name
-             FROM information_schema.columns
-             WHERE SUBSTR(table_name, 1, 7) = 'sample_'
-                AND table_schema = 'qiita'
-                AND column_name IN ('latitude', 'longitude');"""
-    tables_gen = (t[0] for t in conn.execute_fetchall(sql))
+    with TRN:
+        sql = """SELECT DISTINCT table_name
+                 FROM information_schema.columns
+                 WHERE SUBSTR(table_name, 1, 7) = 'sample_'
+                    AND table_schema = 'qiita'
+                    AND column_name IN ('latitude', 'longitude');"""
+        TRN.add(sql)
+        tables_gen = (t[0] for t in TRN.execute_fetchindex())
```

Comment:
I would advocate for a `flatten=[True/False]` parameter added to the fetch calls, but I know y'all don't like changing what's returned through parameters. If we don't use the parameter, do we just want a flatten util function instead? then it's `flatten(TRN.execute_fetchindex())`. Flattening does show up a lot in the codebase, so best to get this sorted out now.